content/en/tracing/trace_pipeline/ingestion_mechanisms.md

```diff
@@ -241,15 +241,34 @@ With Agent version 7.33 and forward, you can configure the error sampler in the
 
 The rare sampler sends a set of rare spans to Datadog. It catches combinations of `env`, `service`, `name`, `resource`, `error.type`, and `http.status` up to 5 traces per second (per Agent). It ensures visibility on low traffic resources when the head-based sampling rate is low.
 
-In Agent version 7.33 and forward, you can disable the rare sampler in the Agent main configuration file (`datadog.yaml`) or with an environment variable:
+**Note**: The rare sampler captures local traces at the Agent level. If the trace is distributed, there is no way to guarantee that the complete trace will be sent to Datadog.
+
+#### Datadog Agent 7.41.0 and higher
+
+By default, the rare sampler is **not enabled**. 
+
+Note: When **enabled**, spans dropped by tracing library rules or custom logic such as `manual.keep` will be evaluated under this sampler.
```

Comment:
Thanks for clarifying, is there a `datadog.yaml` version of this configuration?

(I updated the wording a bit, let me know if it looks better now. It will take several minutes for the preview to update though)